### PR TITLE
2022 Recap Blog

### LINKED ISSUE #1262: [Blog] OpenSearch Project 2022 Recap and What's Next
**Describe the blog post your would like to write**
This blog takes a look back at what the OpenSearch Project accomplished in 2022, and provides readers with some insight into what is to come for the project in 2023. 

**What is the title of the blog post?**
 OpenSearch Project 2022 Recap and What's Next

**Who are the authors?**
@dtaivpp 

**What is the proposed posting date?**
01/19/2023

### PR DESCRIPTION
**Pull request by dtaivpp**
Signed-off-by: David Tippett <dtip@amazon.com>

### Description
This is a short recap of 2022 and a looking ahead to 2023 blog. 
 
### Issues Resolved
Fixes #1262

### Check List
- [X] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the BSD-3-Clause License.

### CODE REVIEW

**Comment by dtaivpp** [comment]
@pajuric  @jhmcintyre could you all look over my keywords/meta description. I think there may be some better ones to include but am not sure what.

File: _posts/2023-01-19-opensearch-project-2022-recap-and-whats-next.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+layout: post
+title:  "OpenSearch Project 2022 recap and what's next"
+authors:
+  - dtaivpp
+  - elifish
+date:   2023-01-19 00:00:00 -0700
+categories:
+  - technical-post
+meta_keywords: "OpenSearch, OpenSearch Dashboards, 2022, Contributors"
+meta_description: "OpenSearch's community has grown signifigantly during 2022. Take a look at what we are doing in 2023 to making building with OpenSearch better!"
```
Comment (jhmcintyre) [inline _posts/2023-01-19-opensearch-project-2022-recap-and-whats-next.md:11]: should read "significantly" and "to make building with"

Comment (pajuric) [reply _posts/2023-01-19-opensearch-project-2022-recap-and-whats-next.md:11]: The keywords are a bit broad. I would suggest these edits: 
OpenSearch roadmap, OpenSearch dashboards, open-source solution for search and analytics, OpenSearch migrations

Comment (pajuric) [reply _posts/2023-01-19-opensearch-project-2022-recap-and-whats-next.md:11]: @dtaivpp - Be sure to updatethe release date on your blog.

**Comment by pajuric** [comment]
Content is finalized and ready to publish.

**Pull request merged**